### PR TITLE
feat(pending indication): adds pending indication

### PR DESCRIPTION
**Pull request by starjardin**
Hi team,

We have just implemented the pending indication. Please have a look at it and leave some comments if you think we can make it better.
Refs: 45e3k5

Thanks,
Honey.

### CODE REVIEW

**Review by Fodilahy-mena — APPROVED** [review]

File: src/components/PendingIndicator/PendingIndicator.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+	title: 'Components/Pending Indicator',
+	component: PendingIndicator,
+} as Meta
+
+const Template: Story<IconType> = (args) => <PendingIndicator {...args} />
+
+export const PendingMedium = Template.bind({})
+PendingMedium.args = {
+	alt: 'PendingMediumIcon',
+	width: '10%',
```
Comment (andrzej-onja) [inline src/components/PendingIndicator/PendingIndicator.stories.tsx:14]: Why % ? I think it should be exact px

File: src/components/PendingIndicator/PendingIndicator.tsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+const ImageStyles = styled.img`
+	height: auto;
+	max-width: ${({ width }) => (width ? width : '50%')};
+`
+export interface IconType {
+	alt: string
+	width: string
+}
+
```
Comment (andrzej-onja) [inline src/components/PendingIndicator/PendingIndicator.tsx:14]: export interface IconType {
	size: "xs" | "s" | "m" | "l"
}

File: src/components/PendingIndicator/PendingIndicator.tsx [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+const PendingIndicator: FC<IconType> = ({ alt, width }) => {
```
Comment (andrzej-onja) [inline src/components/PendingIndicator/PendingIndicator.tsx:15]: function getSize(size: IconSize) 
{
    switch (size) 
    {
        case "xs": return 20;
        case "s": return 40;
        case "m": return 64;
        case "l": return 164;

        default: return 300;
    }    
}

**Review by vahona — APPROVED** [review]

File: src/components/PendingIndicator/PendingIndicator.tsx [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+				return 40
+			case 'm':
+				return 64
+			case 'l':
+				return 164
+
+			default:
+				return 300
+		}
+	}
```
Comment (andrzej-onja) [inline src/components/PendingIndicator/PendingIndicator.tsx:30]: please put getSize function outside of the component (the same file)

**Review by andrzej-onja — APPROVED** [review]

**Pull request merged**